hnn_core/network.py

```diff
@@ -750,6 +753,40 @@ def _instantiate_drives(self, n_trials=1):
                 self.external_drives[
                     drive['name']]['events'].append(event_times)
 
+    def connect_cells(self, src_gids, target_gids, weight, receptor, location,
+                      delay, space_constant):
```

Comment:
I'm almost tempted to suggest that this should be in the constructor of `Network` ... but not sure. Let's think of the API once we know it works